### DISCUSSION TITLE
Report of the 2024 March OSPO Local Meetup in Helsinki

### DISCUSSION THREAD

**Discussion by CsatariGergely** [todogroup/ospology#447] [OSPO Stories]
Our March Meetup was about [Cyber resilience, health and sponsoring of open source projects](https://www.meetup.com/helsinki-ospo-local-meetup-supported-by-todo-group/events/299483548/). The 10 participants who gathered in the offices of [HH Partners](https://www.hhpartners.fi/en/) and enjoyed the food and beverages support from [DoubleOpen](https://www.doubleopen.org/) participated in two in person and one virtual presentation. 

Juan Antonio Osorio from [Stacklok](https://stacklok.com/) explained how to minimise the risk of open source usage by using healthy open source projects. Stacklok provides two tools for this: [Minder](https://stacklok.com/minder) and [Trusty](https://stacklok.com/trusty).
Gergely Csatari from Nokia briefly explained the different ways to support critical open source projects in a presentation based on a [presentation from Kara Sowles in FOSDEM 2024](https://fosdem.org/2024/schedule/event/fosdem-2024-2751-the-state-of-funding-free-open-source-software/).
Ciarán O'Riordan from Open Forum Europe explained the story of EU's Cyber Resilience Act and what will be the next steps. 
We plan to organize the next event in May.

**Comment by anajsana** [comment]
Thank you for sharing @CsatariGergely! do you know if speakers have shared their presentations somewhere? 🙂 Might be useful to also link to this thread!

**Reply by CsatariGergely** [reply]
Sure all presentations are shared in the Meetup page. I will crosslink them here.

**Comment by CsatariGergely** [comment]
Link to the presentations: 

- [Juan Antonio Osorio (Stacklock): Minimise the risk of open source usage by using healthy open source projects](https://speakerdeck.com/jaormx/minimise-the-risk-of-open-source-usage-by-using-healthy-open-source-projects)
- [Gergely Csatari (Nokia): Ways to support critical open source projects](https://drive.google.com/file/d/15njCc2GEIJraDPCnyu99ieRAf50DKPQD/view?usp=sharing)

**Reply by anajsana** [reply]
I think the two tools shared in the 1st presentation, [Minder](https://stacklok.com/minder) and [Trusty](https://stacklok.com/trusty) should be added to the OSPO Landscape in the tooling category: https://landscape.todogroup.org/card-mode?category=sca&grouping=category
